runtime/src/lib.rs

```diff
@@ -159,6 +192,21 @@ pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 1 * HOURS;
 pub const UNIT: Balance = 1_000_000_000_000;
 pub const MILLIUNIT: Balance = 1_000_000_000;
 pub const MICROUNIT: Balance = 1_000_000;
+
+/// The existential deposit. Set to 1/10 of the Connected Relay Chain.
+pub const EXISTENTIAL_DEPOSIT: Balance = MILLIUNIT;
+
+/// We assume that ~5% of the block weight is consumed by `on_initialize` handlers. This is
+/// used to limit the maximal weight of a single extrinsic.
+const AVERAGE_ON_INITIALIZE_RATIO: Perbill = Perbill::from_percent(5);
```

Comment:
in 'master' branch this is 10%, is 5% better for us?

Reply:
I have added details in shared sheet `Runtime Upgrade Values`

Reply:
Raised issue here https://github.com/DataHighway-DHX/node/issues/247